### PR TITLE
build, release-notes: Document 0.8.0 release

### PR DESCRIPTION
**Pull request by rbradford**
Update the build version and the release notes for the 0.8.0 release.

Signed-off-by: Rob Bradford <robert.bradford@intel.com>

### CODE REVIEW

File: release-notes.md [hunk 1]
```diff
@@ -58,6 +66,79 @@
...
+This is non exhaustive list of HTTP API and command line changes:
+
+* All user visible socket parameters are now consistently called `socket`
+  rather than `sock` in some cases.
+* The `ch-remote` tool now shows any error message generated by the VMM
+* The `wce` parameter has been removed from `--disk` as the feature is always
+  offered for negotiaton.
+* `--net` has gained a `host_mac` option that allows the setting of the MAC
+  address for the `tap` device on the host.
+
```
Comment (sameo) [inline release-notes.md:141]: Do we want to add the contributors list here as well?

Comment (rbradford) [reply release-notes.md:142]: I generated it but forgot to add it! Done!

File: release-notes.md [hunk 2]
```diff
@@ -1,3 +1,12 @@
+- [v0.8.0](#v080)
+    - [Experimental Snapshot and Restore Support](#experimental-snapshot-and-restore-support)
+    - [Experimental ARM64 Support](#experimental-arm64-support)
+    - [Support for Using 5-level Paging in Guests](#support-for-using-5-level-paging-in-guests)
+    - [Virtio Device Interrupt Supression for Network Devices](#virtio-device-interrupt-supression-for-network-devices)
```
Comment (likebreath) [inline release-notes.md:5]: Typo: `Supression` -> `Suppression` . There are also a few other same typos in this notes.

File: release-notes.md [hunk 3]
```diff
@@ -58,6 +67,96 @@
...
+
+### Command Line and API Changes
+
+This is non exhaustive list of HTTP API and command line changes:
+
+* All user visible socket parameters are now consistently called `socket`
+  rather than `sock` in some cases.
+* The `ch-remote` tool now shows any error message generated by the VMM
+* The `wce` parameter has been removed from `--disk` as the feature is always
+  offered for negotiaton.
```
Comment (likebreath) [inline release-notes.md:139]: Typo: `negotiaton` -> `negotiatoin`?

File: release-notes.md (hunk 2 above)
Comment (rbradford) [reply release-notes.md]: Thanks.

File: release-notes.md (hunk 3 above)
Comment (rbradford) [reply release-notes.md]: Fixed. Thanks!

**Pull request merged**